sdk/identity/azure-identity-broker/src/main/java/com/azure/identity/broker/implementation/package-info.java

```diff
@@ -0,0 +1,4 @@
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+
```

Comment:
Docs: This needs to be updated to have detailed docs. azure-identity can be used as reference.